addons/mail/static/src/core/thread_service.js

```diff
@@ -1046,12 +1049,8 @@ export class ThreadService {
      */
     clearComposer(composer) {
         composer.attachments.length = 0;
-        composer.textInputContent = "";
-        Object.assign(composer.selection, {
-            start: 0,
-            end: 0,
-            direction: "none",
-        });
+        composer.range = undefined;
+        composer.wysiwygValue = "<p><br></p>";
```

Comment:
so, empty composer is now `"<p><br></p>"`? This looks weird but I guess this is how the web editor works. It might be great to introduce a getter with a meaningful name for the places where you want to check for emptiness 🙂